### ISSUE TITLE
Typos in Io Reference Core/File Code Sample

### ISSUE THREAD

**Issue by mrandrewmills** [stevedekorte/io#372]
I'm new to io, and working through the Guide, Tutorial, and Reference sections of the io language website. I've noticed errors in the first, second, and fourth lines of the code example of the io Reference documentation for Core / File which prevent it from working properly.

```
file = File clone openForUpdating("/tmp/test")
lines = file readLines reverse
file rewind
lines foreach(i, line, file write(line, "\n"))
file close
```
I'm inferring the intent is to open /tmp/test, read the content in by line in reverse order, return to the start of the file, write the reversed lines back into the open file, and then close the file.

```
file := File clone openForUpdating("/tmp/test")
lines := file readLines reverse
file rewind
lines foreach(line, file write(line, "\n"))
file close
```

Am I correct to assume submitting a pull request for /docs/reference/Core/Core/File/index.htm with this correction will eventually lead to the code sample on the website (i.e. http://iolanguage.org/reference/index.html#Core.File) being changed as well?  Or is there another process I need to follow?

Thank you,

Andrew

**Comment by stevedekorte** [comment]
Io's reference docs are generated from comments in the source so the best place to make the fix is in the IoFile.c or maybe File.io (whichever contains the problem). I can pull your fix and try to generate new docs from them, or you could update it in both the source and html file and I can pull and copy the html file to the site. Thanks Andrew!

**Comment by apjanke** [comment]
Looks like this is fixed, in 0a49201ab076d9f4fdcd1e8075c23772f37d0df1. Can this issue be closed?

**Comment by stevedekorte** [comment]
Thanks - I'm leaving it open for now to remind me to push the changes to the site.
